controllers/storagecluster/external_resources.go

```diff
@@ -268,6 +274,11 @@ func (obj *ocsExternalResources) ensureCreated(r *StorageClusterReconciler, inst
 			}
 			externalOCSResources[instance.UID] = externalConfig
 		}
+
+		if err := r.checkSCsAndCallCreateDefaultClaim(instance); err != nil {
+			return reconcile.Result{}, err
+		}
```

Comment:
Is there a reason we need to do this here and not where we created all of the other external resources?
In addition, if this needs to be here,  is there a reason why we are keeping the externalClusterClient open during this operation? I don't think it is using it.

Reply:
> Is there a reason we need to do this here and not where we created all of the other external resources? 

can you point out where you are suggesting? 

>In addition, if this needs to be here, is there a reason why we are keeping the externalClusterClient open during this operation? I don't think it is using it.

moved

Reply:
@nb-ohad  ready to mark resolved?